samples/nfc/system_off/prj.conf

```diff
@@ -15,4 +15,3 @@ CONFIG_NFC_NDEF_MSG=y
 CONFIG_NFC_NDEF_RECORD=y
 CONFIG_NFC_NDEF_TEXT_RECORD=y
 CONFIG_POWEROFF=y
-CONFIG_PM_DEVICE=y
```

Comment:
Previously you had PM_DEVICE enabled for nrf52 and nrf53. Now it won't be enabled anymore?